common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -98,7 +98,10 @@ export const ButtonIconWrapper = styled(
   className: classNames({
     'flex-inline': true,
   }),
-}))<ButtonIconWrapperAttrsProps>``;
+}))<ButtonIconWrapperAttrsProps>`
+  // Prevent icon within .spaced-text parent having top margin
```

Comment:
Stops this happening

![image](https://user-images.githubusercontent.com/1394592/186151338-19343066-628a-4ec6-89a3-c1ed8ea0cb0f.png)